site_root/input/resources/extensions/StructureDefinition-age-at-event.json

```diff
@@ -0,0 +1,72 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "age-at-event",
+  "url": "http://fhir.ncpi-project-forge.io/StructureDefinition/age-at-event",
+  "version": "0.1.0",
+  "name": "age-at-event",
+  "title": "Age At Event",
+  "status": "draft",
+  "description": "Age (since birth) of Patient at time of event.",
+  "fhirVersion": "4.0.0",
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "Patient"
+    },
+    {
+      "type": "element",
+      "expression": "Observation"
+    },
+    {
+      "type": "element",
+      "expression": "Condition"
+    },
+    {
+      "type": "element",
+      "expression": "Specimen"
+    }
+  ],
```

Comment:
This part can be taken out if we do not want to bind this extension to specific profiles.